.github/workflows/release.yml

```diff
@@ -33,20 +34,30 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - name: Push release tag
+    - name: Push lib release tag
+      if: "! startsWith(github.event.head_commit.message, 'release: cargo-binstall v')"
```

Comment:
```suggestion
      if: "! startsWith(github.event.head_commit.message, 'release: binstall v')"
```

Reply:
Nope, that's intentional. We're matching everything _but_ the CLI (which has a crate name of `cargo-binstall`).

Reply:
Oh thanks for pointing that out!
I missed the `!` after `if:`.